pkgs/applications/audio/surge-XT/default.nix

```diff
@@ -1,30 +1,70 @@
-{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, cairo, libxkbcommon, xcbutilcursor, xcbutilkeysyms, xcbutil, libXrandr, libXinerama, libXcursor, alsa-lib, libjack2
-}:
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, cairo, libxkbcommon
+, xcbutilcursor, xcbutilkeysyms, xcbutil, libXrandr, libXinerama, libXcursor
+, alsa-lib, libjack2, lv2 }:
 
-stdenv.mkDerivation rec {
+let
+  juce-lv2 = stdenv.mkDerivation rec {
+    name = "JUCE-lv2";
+
+    src = fetchFromGitHub {
+      owner = "lv2-porting-project";
+      repo = "JUCE";
+      rev = "f7eb3b4681c1fb4735bcd388d7a5ed10ccd24aba";
+      sha256 = "sha256-jwWDB0XXaKkHJDNOSnhV4tDd+2lt61Qp8gFwXDuY63c=";
+    };
+    installPhase = ''
+      cp -r . $out
+    '';
+  };
+
+in stdenv.mkDerivation rec {
   pname = "surge-XT";
-  version = "unstable-2021-11-07";
+  version = "unstable-2021-12-07";
 
   src = fetchFromGitHub {
     owner = "surge-synthesizer";
     repo = "surge";
-    rev = "ed93833eb44b177c977e3a7b878ffdd9bf9f24e5";
-    sha256 = "0b164659ksl6h5nn7jja5zccx2mwzibqs6b7hg8l98gpcy9fi5r2";
+    rev = "c61ae7f2e1e8ecfc646030618bd3c4439d3ddf79";
+    sha256 = "sha256-R8qGDxJffYddeDjEtoIe0gB+WMwCEyvoAAhaxPyM/SA=";
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [ cmake pkg-config ];
-  buildInputs = [ cairo libxkbcommon xcbutilcursor xcbutilkeysyms xcbutil libXrandr libXinerama libXcursor alsa-lib libjack2 ];
+  buildInputs = [
+    cairo
+    libxkbcommon
+    xcbutilcursor
+    xcbutilkeysyms
+    xcbutil
+    libXrandr
+    libXinerama
+    libXcursor
+    alsa-lib
+    libjack2
+    lv2
+  ];
+
+  cmakeFlags = [
+    "-Bbuild"
+    "-DJUCE_SUPPORTS_LV2=True"
+    "-DSURGE_JUCE_PATH=${juce-lv2}"
+  ];
+
+  enableParallelBuilding = true;
+
+  buildPhase = ''
+    cmake --build build --target surge-xt_LV2 surge-fx_LV2 surge-xt_Packaged surge-fx_Packaged --parallel
+  '';
```

Comment:
The way this is written does not value the user's configured `nix.buildCores`, or any of your `cmakeFlags`. I was abit started when my system suddenly slowed down, because it consumed all 48 cores instead of just the 6 I configured Nix to use by default. Please try moving the targets into `cmakeFlags`.
```suggestion
```